locales/nl-NL.json

```diff
@@ -28,14 +28,20 @@
     "muted_users": "Gedempte gebruikers",
     "muting": "Gedempt",
     "mutuals": "Gedeeld",
+    "notifications_on_post_disable": "Stop met melden wanneer {username} post",
+    "notifications_on_post_enable": "Meld me wanneer {username} post",
```

Comment:
@mxcop Currently "notifications" has been translated as "notificaties" in other places. Would it be worth changing those to "Meldingen"? That mimics the bird site, so don't know if that's desirable or not.

Reply:
I do think that "Meldingen" sounds a bit better. So imo you can change that if you want :D